metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/AspectWithMetadata.pdl

```diff
@@ -0,0 +1,8 @@
+namespace com.linkedin.metadata.aspect
+
+import com.linkedin.common.AuditStamp
+
+record AspectWithMetadata {
```

Comment:
Can we call this "VersionedAspect"?

Reply:
I prefer names without things like "with" or "and"

Reply:
`VersionedAspect` is a bit too specific- we may want to add more metadata later. I can think about a different name

Reply:
I'm not so sure - I think versioned aspect conveys much more detail and I'm not sure what other metadata we'll want here

Reply:
fyi renamed to VersionedAspect :)